e2e/cypress/integration/channel_settings/channel_name_validations_spec.js

```diff
@@ -68,4 +72,38 @@ describe('Channel routing', () => {
         // # close the create channel modal
         cy.findByText('Cancel').click();
     });
+
+    it('MM-T883 Channel URL validation for spaces between characters', () => {
+        // # In a test channel, click the "v" to the right of the channel name in the header
+        cy.findByText(`${testChannel.display_name}`).click();
+        cy.get('#channelHeaderDropdownIcon').click();
+
+        // # Select "Rename Channel"
+        cy.findByText('Rename Channel').click();
+
+        // # Change the channel name to {26 alphanumeric characters}[insert 2 spaces]{26 alphanumeric characters}
+        //   i.e. a total of 54 characters separated by 2 spaces
+        cy.get('#display_name').clear().type(`${Cypress._.repeat('a', 26)}${Cypress._.repeat(' ', 2)}${Cypress._.repeat('b', 26)}`);
```

Comment:
Please assign those repeated characters into variable and reuse as needed. Also, you may use the random ID helper function in this case to come up with alphanumeric characters,
```javascript
import {getRandomId} from '../../utils';
...
const firstWord = getRandomId(26);
const secondWord = getRandomId(26);
```